exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,11 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var number = 100;
+ 
+var percentStudents = (numberOfStudents / 100) * number;
+var  percentMentors= (numberOfMentors / 100) * number;
+console.log(percentStudents + "%")
+console.log(percentMentors + "%")
+
+ 
+
```

Comment:
in this exercise should 
Percentage student: 65% not 15%
and Percentage mentors: 35%  not 8%

Reply:
greet job ,,
: Percentage = (value/total value) x 100